JSDOMParser.js

```diff
@@ -472,7 +524,27 @@
 
     nodeName: "#text",
     nodeType: Node.TEXT_NODE,
-    textContent: ""
+    get textContent() {
+      if (typeof this._textContent === "undefined") {
+        this._textContent = decodeHTML(this._innerHTML || "");
```

Comment:
Nit: `this._textContent = decodeHTML(this._innerHTML)  || "";` might be some ns faster